joomla/joomla-cms
# [3.4] Fix the new Updatesites enable and disable messages

### How to test
1. Access the updatesites view
2. disable one update server
3. you get `1 extension successfully disabled.`
4. enable one update server
5. you get `1 extension successfully enabled.`
6. apply the patch
7. retest 2 and 4
8. you get  `1 update site successfully disabled.` and  `1 update site successfully enabled.`
9. also test with 2 ore more update sites that you enable or disable
10. you get something like this: `2 update sites successfully disabled.` and  `2 update sites successfully enabled.`

Please have a look here @nikosdion I think this is what you expect or I'm wrong here? Thanks
